README.md

```diff
@@ -78,11 +78,31 @@ You are ready to save and load data using `Snack`.
 ```python
 snack.set(Person("1", "John"))
 # 'Person-1'
-snack.get(Person, ["1"])
+entity = snack.get(Person, ["1"])
```

Comment:
I added it here to follow the same `pattern` as below (this declaration one line below was already there, I didnt change it). Also, for me it makes more sense since usually `get()` methods return something, so `entity = snack.get(Person, ["1"])`  is more descriptive in my opinion. But it's only readme, I can change it if you guys have other ideas, we can agree on one convention